db/item_db2.conf

```diff
@@ -70,7 +70,7 @@ item_db: (
 		Taekwon: true/false           (boolean, defaults to false)
 		Star_Gladiator: true/false    (boolean, defaults to false)
 		Soul_Linker: true/false       (boolean, defaults to false)
-		Gungslinger: true/false       (boolean, defaults to false)
+		Gunslinger: true/false       (boolean, defaults to false)
```

Comment:
Add a space after true/false, so that the whitespace is consistent with all other lines